cmd/gsw/ggTools.go

```diff
@@ -64,24 +64,27 @@ func RunGgTools() {
 		log.Printf("Reading: %s\n", inFile)
 		switch true {
 		case chain.IsChainFile(inFile):
-			log.Printf("chain input detected!\n")
+			log.Printf("Input chain detected...\n")
```

Comment:
The changes to this file seem like they could be separated from many of the other changes in the pull request.  Maybe this pull request could be broken into a handful of smaller ones?